docs/source/tools.rst

```diff
@@ -172,7 +172,7 @@ Then, we display the result of `Circuit Analyzer` via ``pdisplay``.
 
 >>> pcvl.pdisplay(ca)
 
-.. figure:: _static/img/CircuitAnalyserHOM.png
```

Comment:
I think you should rollback this change, else the image link is broken

Reply:
No because I also renamed the png file

Reply:
Right, sorry